### PR TITLE
File structure.

### PR DESCRIPTION
**Pull request by moustf**
- Created the file structure.

### CODE REVIEW

**Review by momenmarwan — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    'plugin:@typescript-eslint/recommended',
+    // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors
+    // as ESLint errors. Make sure this is always the last configuration in the extends array.
+    'plugin:prettier/recommended',
+  ],
+  rules: {
+    // Place to specify ESLint rules. Can be used to overwrite rules specified
+    // from the extended configs
+    // e.g. "@typescript-eslint/explicit-function-return-type": "off",
+    '@typescript-eslint/no-explicit-any': 'off',
```
Comment (momenmarwan) [inline .eslintrc.js:24]: why you turn off the any of typescript what the point of typescript

File: server/index.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { app } from './app';
+
+const port = app.get('port');
+
+app.listen(port, () => {
```
Comment (momenmarwan) [inline server/index.ts:5]: this function doesn't return anything why we don't use the void keyword

File: server/app.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import express, { Request, Response } from 'express';
+import compression from 'compression';
+import morgan from 'morgan';
+import cookieParser from 'cookie-parser';
+import cors from 'cors';
+
+export const app = express();
```
Comment (momenmarwan) [inline server/app.ts:7]: we should specify the type of app maybe we use 
 const app: Application = express();
or 
const app: Express = express();
I don't know what is better

**Review by momenmarwan — CHANGES_REQUESTED** [review]

File: client/src/App.tsx
Comment (Farah202324) [inline client/src/App.tsx:1]: import './App.css';

function App(): React.ReactElement {
  return <div>hello</div>;

}

export default App;

By adding : React.ReactElement after the function name, you explicitly state that the App function returns a ReactElement. This helps TypeScript analyze and enforce type correctness in your code.

File: .eslintrc.js
Comment (Farah202324) [inline .eslintrc.js:1]: any is not suppose to be allowed

**Pull request closed without merging**